api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -493,26 +496,35 @@ public void testSubmitDataUseAgreement_success() {
     assertThat(dbUserDataUseAgreement.getUserFamilyName()).isEqualTo(dbUser.getFamilyName());
     assertThat(dbUserDataUseAgreement.getUserGivenName()).isEqualTo(dbUser.getGivenName());
     assertThat(dbUserDataUseAgreement.getUserInitials()).isEqualTo(duaInitials);
-    assertThat(dbUserDataUseAgreement.getDataUseAgreementSignedVersion()).isEqualTo(DUA_VERSION);
+    assertThat(dbUserDataUseAgreement.getDataUseAgreementSignedVersion()).isEqualTo(DUCC_VERSION);
   }
 
   @Test
-  public void testSubmitDataUseAgreement_wrongVersion() {
+  public void testSubmitDataUseAgreement_wrongVersion_older() {
     assertThrows(
         BadRequestException.class,
         () -> {
           createAccountAndDbUserWithAffiliation();
           String duaInitials = "NIH";
-          profileController.submitDataUseAgreement(DUA_VERSION - 1, duaInitials);
+          profileController.submitDataUseAgreement(DUCC_VERSION - 1, duaInitials);
         });
   }
 
+  // not really a use case for this, but shows we need an exact match
+
   @Test
-  public void test_outdatedDataUseAgreement() {
-    // force version number to 2 instead of 3
-    config.featureFlags.enableV3DataUserCodeOfConduct = false;
-    final int previousDuaVersion = DUA_VERSION - 1;
+  public void testSubmitDataUseAgreement_wrongVersion_newer() {
+    assertThrows(
+        BadRequestException.class,
+        () -> {
+          createAccountAndDbUserWithAffiliation();
+          String duaInitials = "NIH";
+          profileController.submitDataUseAgreement(DUCC_VERSION + 1, duaInitials);
+        });
+  }
 
+  @Test
+  public void test_outdatedDataUseAgreement() {
```

Comment:
so current version is DUCC_VERSION and this test tests against current version as well as DUCC_VERSION +1 ?

Reply:
This test updates the current version, so the old-current version is no longer current.

OK that is super confusing.  I'll change some words.